src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java

```diff
@@ -67,10 +68,10 @@
 import java.util.Set;
 
 public abstract sealed class AbstractLinker implements Linker permits LinuxAArch64Linker, MacOsAArch64Linker,
-                                                                      AixPPC64Linker,
                                                                       SysVx64Linker, WindowsAArch64Linker,
-                                                                      Windowsx64Linker, LinuxPPC64leLinker,
-                                                                      LinuxRISCV64Linker, LinuxS390Linker,
+                                                                      Windowsx64Linker, AixPPC64Linker,
+                                                                      LinuxPPC64leLinker, LinuxRISCV64Linker,
+                                                                      LinuxS390Linker, ZosS390Linker,
                                                                       FallbackLinker {
```

Comment:
Please simplify the change relative to upstream.
```java
public abstract sealed class AbstractLinker implements Linker permits LinuxAArch64Linker, MacOsAArch64Linker,
                                                                      AixPPC64Linker,
                                                                      SysVx64Linker, WindowsAArch64Linker,
                                                                      Windowsx64Linker, LinuxPPC64leLinker,
                                                                      LinuxRISCV64Linker, LinuxS390Linker,
                                                                      ZosS390Linker,
                                                                      FallbackLinker {
```
Written this way, we're simply adding two lines (one for AIX and the other for z/OS).

Reply:
Fixed as suggested above.